src/commands/functions-deletegcfartifacts.ts

```diff
@@ -0,0 +1,47 @@
+import { Command } from "../command";
+import * as utils from "../utils";
+import * as getProjectId from "../getProjectId";
+import { listGCFArtifacts, deleteGCFArtifacts } from "../deploy/functions/containerCleaner";
+import { promptOnce } from "../prompt";
+import { requirePermissions } from "../requirePermissions";
+
+function getConfirmationMessage(artifacts: Set<string>): string {
```

Comment:
@inlined  I'm okay with listing out the full GCR path since that would allow the user to easily check for artifacts without looking up UUIDs. This requirement came from a discussion with @kaibolay on the API proposal. Kai ultimately approved the proposal, so we should probably get his opinion on this change.

Reply:
I really like listing the full GCR path(s).

Reply:
I guess it's time to disagree yet commit =D

Reply:
Sorry, I should have been clearer. I don't think we disagree. 

What I meant to say is that I really like displaying the full directory path (without the UUIDs of the individual images).

Maybe we could display the number of images in those directories; e.g.:

```
You are about to delete all images in the following directories:

us.gcr.io/myproject/gcf/us-central1 (3 images)
us.gcr.io/myproject/gcf/us-west1 (1 image)

Are you sure?
```

But that's only a nice to have. The directory paths alone contain 99.9% of the important information.

Reply:
I've changed it to display the full directory path. I explored adding in the image counts but that is a recursive operation that calls the docker API for each directory and sub directories. We would probably get rate limited and could potently run into a similar issue as [this](https://github.com/firebase/firebase-tools/issues/3639) where the command fails.

Thomas also pointed out that some customers that are blocked on this issue have a massive amount of images. If those customers were to run this command then they'd likely not make much progress waiting for the list to load or get fully blocked when running this command.